### PR TITLE
Adds dart gun as Trader-Exclusive syndicate item.

### PR DESCRIPTION
**Pull request by gurfan**
Costs 16 TC, starts loaded with the usual stoxin/chloral that raiders get. Additional cartridges can be purchased for 2 TC each. Both are only available to traders.

:cl:
 * rscadd: Traders can now purchase dart guns and dart cartridges from their syndicate uplinks.

### CODE REVIEW

File: code/datums/uplink_item.dm [hunk 1]
```diff
@@ -1014,3 +1014,22 @@ var/list/uplink_items = list()
...
+	name = "Chemical Dart Gun"
+	desc = "A staple in vox weaponry. This dart gun starts loaded with darts containing sleep toxin and chloral hydrate. The beaker inside can be swapped out to create your own deadly mixes."
+	item = /obj/item/weapon/gun/dartgun/vox/raider
+	cost = 12
+	jobs_exclusive = list("Trader")
+
+/datum/uplink_item/jobspecific/trader/dart_cartridge
+	name = "Dart Cartridge"
+	desc = "A spare cartridge to refill your dart gun."
+	item = /obj/item/weapon/dart_cartridge/
```
Comment (jknpj) [inline code/datums/uplink_item.dm]: wild /

**Comment by TGFH** [comment]
Way too strong. The dart gun is just a syringe gun that ignores hardsuits. AKA a ranged multi shot parapen for maximum silent shuffling. It's fine for raiders really because they're intended as a largely nonlethal antag and once raiders are detected they're a known quantity. I really can't imagine that traitors would use this for anything other than just silent murderbone.

**Comment by gurfan** [comment]
The item is only available to traders so normal crewmembers wouldn't be able to murderbone with it unless they somehow get their hands on it. It's price is the same as the ebow which is already the primo murderbone tool but I can increase it

**Comment by coronasaurus** [comment]
s-s-s-silent shuffle

**Comment by gurfan** [comment]
price 12 -> 16, conflicts fixed

File: code/datums/uplink_item.dm [hunk 2]
```diff
@@ -1018,9 +1018,24 @@ var/list/uplink_items = list()
 /datum/uplink_item/jobspecific/trader
 	category = "Trader Specials"
 
+/datum/uplink_item/jobspecific/trader/dartgun
+	name = "Chemical Dart Gun"
+	desc = "A staple in vox weaponry. This dart gun starts loaded with darts containing sleep toxin and chloral hydrate. The beaker inside can be swapped out to create your own deadly mixes."
+	item = /obj/item/weapon/gun/dartgun/vox/raider
+	cost = 12
```
Comment (Zth--) [inline code/datums/uplink_item.dm]: this is not 16

**Pull request merged**